### DISCUSSION TITLE
coa_tools2 is now available up to blender 4.3

### DISCUSSION THREAD

**Discussion by Aodaruma** [Aodaruma/coa_tools2#58] [Announcements]
## tldr

The main updates include modifications to make the existing coa_tools compatible with version 4.3.0, as well as significant UI and functionality improvements to the Photoshop script. Additionally, the Blender addon has some new features.
On the other hand, certain features, such as JSON export for Godot, are not functioning correctly. (Please refer to [issues](https://github.com/Aodaruma/coa_tools2/issues)for detailed bug reports.)

## What's Changed in detail ( written by github copilot)
* Create Pipfiles and Rewrite README.md by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/5
* add information, change an author and LICENSE information in coa_tools, and add Development wiki in README.md by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/7
* add old coa_tools version converter by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/12
* Add sprite_import_export_scale by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/14
* Feature improve uv and edit mesh #15 by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/16
* fixed import: added lost property into COATools2Preferences by @tozpeak in https://github.com/Aodaruma/coa_tools2/pull/21
* Add ko_fi by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/24
* Hotfix krita import error - resolved #25 by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/27
* add github sponsors by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/29
* Merge develop into Master by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/30
* Delete duplicated definition of sprite_import_export_scale by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/31
* add tex_node extension = clip by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/32
* Add instructions how to install Krita plugin - Resolves #37 by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/38
* add checkbox into a preference panel for coatools updater - Resolves #22 by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/39
* fix conditions showing the category - Resolves #36 by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/40
* merge Develop into Master by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/41
* improve installation instructions and add iamges - Resolves #26 by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/42
* Update README.md by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/43
* Fix error convert old coatools by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/44
* update fake-bpy-modile and some code fix for 4.0.0 by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/46
* Hotfix for importing error in Blender 4.0.0 (node types changed in 4.0.0 update) by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/49
* Fix codes to support bone group update in Blende 4.0.0 by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/50
* Fix bpy.ops.wm.tool_set_by_id error in Blender 3.6 or below by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/51
* Feature improvements Photoshop Exporter #17 by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/54
* Sync commits from upstream/master by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/56
* Upgrade coa_tools2 to blender 4.3 #53 by @Aodaruma in https://github.com/Aodaruma/coa_tools2/pull/57

## New Contributors
* @Aodaruma made their first contribution in https://github.com/Aodaruma/coa_tools2/pull/5
* @tozpeak made their first contribution in https://github.com/Aodaruma/coa_tools2/pull/21

**Full Changelog**: https://github.com/Aodaruma/coa_tools2/compare/v1.0.0...v2.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Aodaruma/coa_tools2/releases/tag/v2.0.0'>coa_tools2 is now available up to blender 4.3</a>.</em>
